### PR TITLE
go:mod: runtime-spec v1.1.0-rc.1; add docs/spec-conformance.md

### PR DESCRIPTION
**Pull request by AkihiroSuda**
runc now implements the [OCI Runtime Spec v1.1.0-rc.1](https://github.com/opencontainers/runtime-spec/tree/v1.1.0-rc.1) for the `linux` platform.

The following features are not implemented yet:

Spec version | Feature                                  | PR
-------------|------------------------------------------|----------------------------------------------------------
v1.0.0       | `SCMP_ARCH_PARISC`                       | Unplanned, due to lack of users
v1.0.0       | `SCMP_ARCH_PARISC64`                     | Unplanned, due to lack of users
v1.0.2       | `.linux.personality`                     | [#3126](https://github.com/opencontainers/runc/pull/3126)
v1.1.0-rc.1  | `.linux.resources.cpu.burst`             | [#3205](https://github.com/opencontainers/runc/pull/3205)
v1.1.0-rc.1  | `.domainname`                            | [#3600](https://github.com/opencontainers/runc/pull/3600)
v1.1.0-rc.1  | `.[]mounts.uidMappings`                  | [#3717](https://github.com/opencontainers/runc/pull/3717)
v1.1.0-rc.1  | `.[]mounts.gidMappings`                  | [#3717](https://github.com/opencontainers/runc/pull/3717)
v1.1.0-rc.1  | `SECCOMP_FILTER_FLAG_WAIT_KILLABLE_RECV` | TODO

### CODE REVIEW

**Comment by AkihiroSuda** [comment]
Let's release runc v1.2.0-rc.1 after merging this
@opencontainers/runc-maintainers

**Review by kolyshkin — APPROVED** [review]
LGTM

**Comment by AkihiroSuda** [comment]
Rebased, as https://github.com/opencontainers/runc/pull/3377 was merged

File: docs/spec-conformance.md
Comment (rata) [inline docs/spec-conformance.md:15]: As I mentioned in that PR, I think that approach can't work (can't create an idmap mount within a userns). I opened another PR that works: https://github.com/opencontainers/runc/pull/3717

Can we mention #3717 here?

Comment (AkihiroSuda) [reply docs/spec-conformance.md:15]: updated

**Review by thaJeztah — APPROVED** [review]
LGTM

**Comment by AkihiroSuda** [comment]
Rebased

**Comment by AkihiroSuda** [comment]
Rerebased

**Comment by kolyshkin** [comment]
```
=== RUN   TestPodSkipDevicesUpdate
    systemd_test.go:114: container stderr not empty: /bin/basename: missing operand
        Try '/bin/basename --help' for more information.
--- FAIL: TestPodSkipDevicesUpdate (0.14s)
```

Hmm, I have absolutely no idea where the "basename" is coming from in this test. Most probably it's from some bashrc on the testing host (in this case we run bash from the host inside the cgroup, see https://github.com/opencontainers/runc/blob/5c2a1a01912b13ca501a7b81672a1fe10e33a647/libcontainer/cgroups/devices/systemd_test.go#L162). We can maybe add `--norc` to work around this, but in general this should not ever happen.

**Comment by AkihiroSuda** [comment]
Restarted the CI and it is now green

**Comment by rata** [comment]
> **Update:** ah, this is centos-stream-9, perhaps something was broken in there.

I've seen this error yesterday too :-/  I completely forgot about it.

**Comment by AkihiroSuda** [comment]
@kolyshkin @thaJeztah Lost your LGTMs after rebasing, could you LGTM again 🙏

**Review by thaJeztah — APPROVED** [review]
LGTM

**Review by kolyshkin — APPROVED** [review]
still LGTM

**Pull request merged**
